File: tapas_borne/src/main/java/com/mycompany/bornetapashouse/DTO/TablesDTO.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.bornetapashouse.DTO;

/**
 *
 * @author PC
 */
public class TablesDTO {
    private int id;

    // Constructeur avec tous les champs
    public TablesDTO(int id) {
        this.id = id;
    }

    // Getters et setters
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return "TablesDTO{" +
                "id=" + id +
                '}';
    }
}

File: tapas_webservice/src/DTO/TablesDTO.php
<?php
namespace App\DTO;

class TablesDTO implements \JsonSerializable{
    private $id;

    public function setid($newid){
        $this->id= $newid;
    }
    public function getid(){
        return $this->id;
    }

    public function jsonSerialize() {
		return array(
			'id' => $this->id,
		);
	}
}